### DISCUSSION TITLE
Dremel DigiLab 3d45 printer?

### DISCUSSION THREAD

**Discussion by WOWElectronics** [smartavionics/Cura#100] [General]
Can the armhf version of cura make gcode for the 3d45 printer?

**Comment by smartavionics** [comment]
It appears that people have been using that printer with UM Cura, a quick search took me to https://github.com/Xcessity/CuraDremelSupport/ which looks useful. Step 2 of their installation says copy their files into the UM file tree. For my ARM AppImages you can't do that but, alternatively, once you have run the AppImage at least once, you will have files in ~/.local/share/cura/master and so you can copy the CuraDremelSupport files into the appropriate directories in there. Hope this helps.
